Automattic/wp-calypso
# Switch off the segmented flow in favor of the original onboarding flow

Related to p1724083895924549-slack-C02T4NVL4JJ.

## Proposed Changes

This makes the original unsegmented onboarding flow the main flow again.

## Why are these changes being made?

Segmented flows didn't perform better.

## Testing Instructions

Please test thoroughly.

1. Go through the onboarding flow by visiting /start.
2. Make a site with a free domain and a free plan.
3. Make a site with a free domain and a paid plan.
4. When picking a paid domain, the free plan should cease being a column, and should be in the subheader.
     - When clicking the CTA in the subheader, you should see a modal "A paid plan is required for your domain."
     - Please try all three options in the modal by going through the flow 3 times.
5. Pick a paid domain and a paid plan. You should land in checkout
